src/pages/ReportDetailsPage.tsx

```diff
@@ -79,7 +79,7 @@ function ReportDetailsPage({policies, report, session, personalDetails}: ReportD
     const {translate} = useLocalize();
     const {isOffline} = useNetwork();
     const styles = useThemeStyles();
-    const [parentReport] = useOnyx(`${ONYXKEYS.COLLECTION.REPORT}${report.parentReportID ?? '-1'}`);
+    const [parentReport] = useOnyx(`${ONYXKEYS.COLLECTION.REPORT}${report.parentReportID || '-1'}`);
```

Comment:
Right, thank you!

Reply:
Can we do this instead of disabling the rule?
```suggestion
    const parentReportID = report.parentReportID ?? '-1';
    const [parentReport] = useOnyx(`${ONYXKEYS.COLLECTION.REPORT}${parentReportID === '-1' ? parentReportID : report.parentReportID}`);

```

Reply:
Can do, but I think it has to be `parentReportID === '' ? parentReportID : report.parentReportID`

Reply:
Okay maybe this is indeed becoming more confusing to read than just dismissing that stupid rule. 🙄 

My bad! 😅  - let's just go back to dismissing the rule I apologize.

Reply:
It would work I think, but getting headache from it so happy to use the rule exception